docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
```

Comment:
Let us have an appropriate readme file. This is the default one generated by docusaurus.